README.md

```diff
@@ -26,6 +26,13 @@ Mapbox has built one of the best and most amazing OSS ecosystems. A key componen
 ## Donations
 If you or your organisation has seen value from Maputnik, please consider donating at <https://maputnik.github.io/donate>
 
+## Using
```

Comment:
There are already two ways mentioned in the readme how the editor could be used (online at https://maputnik.github.io/editor/ and CLI). I think it's good to have this new section _Using_ and we should move the other two options in here. I would like to keep the order: 
1. online editor
2. CLI
3. docker

Reply:
Makes sense about grouping them together.